CS3219-AY2223S1/cs3219-project-ay2223s1-g12
# 142 refresh question

fixes #142 

### Implemented functionality to refresh question in the room page 
- Frontend: added refresh icon and modal check for confirmation 
- Backend: handled socket.io logic (client emits `refresh-question` to indicate a refresh event and server emits `update-question` with the new question as an argument)

### To test
- Find a match and enter a room
- Inside the room, click on the refresh icon next to question title 
- Click on `yes` when the modal appears
- The question should be changed. Check that the question change is also updated for the other user
